src/fugw/solvers/utils.py

```diff
@@ -474,7 +473,14 @@ def solver_sinkhorn_stabilized(
     tau = 1e3  # Threshold for stabilization
 
     def get_K(alpha, beta):
-        return ((alpha[:, None] + beta[None, :] - cost) / eps).exp()
+        return (-(cost - alpha[:, None] - beta[None, :]) / eps).exp()
+
+    def get_Gamma(alpha, beta, u, v):
```

Comment:
It's good to include short docstrings as it is not obvious what Gamma represents.